p2p_wallet/Scenes/Main/Receive/View/InstructionsReceiveView.swift

```diff
@@ -12,57 +12,84 @@ struct InstructionsReceiveView: View {
-            HStack(alignment: .top, spacing: InstructionsReceiveView.textHorizontalSpacing) {
-                // Hidden element, to save space
-                Text("0")
-                    .frame(width: InstructionsReceiveView.numberSize, height: InstructionsReceiveView.numberSize)
-                    .opacity(0)
-                Text(item.tip)
-                    .apply(style: .text4)
-                    .foregroundColor(Color(Asset.Colors.mountain.color))
-                    .multilineTextAlignment(.leading)
-                    .padding(.bottom, 16)
-                
-            }
+            Spacer()
         }
-        .padding(.top, 24)
+        .padding(.vertical, 24)
         .padding(.horizontal, 15)
         .background(Color(Asset.Colors.snow.color))
         .cornerRadius(radius: 16, corners: .allCorners)
+        .fixedSize(horizontal: false, vertical: true)
     }
+
+    func instructionCell(instruction: (String, (String, String))) -> some View {
+        HStack(alignment: .top, spacing: InstructionsReceiveView.textHorizontalSpacing) {
+            VStack(alignment: .center, spacing: 2) {
+                Text(instruction.0)
+                    .fontWeight(.semibold)
+                    .apply(style: .label1)
+                    .foregroundColor(Color(Asset.Colors.silver.color))
+                    .frame(
+                        width: InstructionsReceiveView.numberSize,
+                        height: InstructionsReceiveView.numberSize
+                    )
+                    .overlay(
+                        RoundedRectangle(cornerRadius: 10.5)
+                            .stroke(Color(Asset.Colors.silver.color), lineWidth: 1.5)
+                    )
+
+                Color(Asset.Colors.silver.color)
+                    .frame(width: 1)
+                    .padding(.bottom, 2)
+                    .if(instruction.0 == item.instructions.last?.0) { view in
+                        view.opacity(0)
+                    }
+            }
+
+            VStack(alignment: .leading, spacing: 4) {
+                Text(instruction.1.0)
+                    .foregroundColor(Color(Asset.Colors.night.color))
+                    .apply(style: .text3)
+                    .multilineTextAlignment(.leading)
+                Text(instruction.1.1)
+                    .foregroundColor(Color(Asset.Colors.mountain.color))
+                    .apply(style: .text4)
+                    .multilineTextAlignment(.leading)
+            }
+            .padding(.bottom, instruction.0 == item.instructions.last?.0 ? 0 : 24)
+        }
+    }
+
 }
 
 struct InstructionsReceiveCellView_Previews: PreviewProvider {
     static var previews: some View {
         VStack(alignment: .leading) {
-            InstructionsReceiveView(item: .init(instructions: [("1", "We bridge to Solana"), ("2", "Send USDC to your Ethereum account"), ("3", "text3")], tip: "You need only to sign a transaction"))
+            InstructionsReceiveView(item: .init(
```

Comment:
Use certain name instead of .init
.init decreases readability and increases compilation time

Reply:
It's preview :)